src/api/spec/factories/packages.rb

```diff
@@ -5,7 +5,9 @@
     title { Faker::Book.title }
     description { Faker::Lorem.sentence }
 
-    after(:create, &:write_to_backend)
```

Comment:
btw: henne, in 245dfbf7b97 you added this, but the model's after_save hook already makes sure this is called. So it's not needed and just repeats the function (which obviously is meaningless in the VCR case). I just spotted it while adding the after(:build) hook

Reply:
If you say so :cowboy_hat_face: